examples/guestbook/README.md

```diff
@@ -31,31 +37,31 @@ Use the file `examples/guestbook/redis-master.json` which describes a single pod
         "image": "dockerfile/redis",
         "cpu": 100,
         "ports": [{
-          "containerPort": 6379,
-          "hostPort": 6379
+          "containerPort": 6379,   # containerPort: Where traffic to redis ultimately is routed to.
+          "hostPort": 6379         # hostPort: The port on the host itself, that the service proxy sends traffic to.
```

Comment:
Please just remove hostPort since it is not needed for the example to work and makes things less clear.